de.dentrassi.pm.aspect.mvnosgi/src/de/dentrassi/pm/aspect/mvnosgi/internal/Constants.java

```diff
@@ -0,0 +1,10 @@
+package de.dentrassi.pm.aspect.mvnosgi.internal;
+
+public class Constants
+{
+    public static final String POM_FILE_NAME = "mvnosgi.pom";
```

Comment:
True ... and I made a change for that :wink: 

But you are right. Giving it a more descriptive name is better when using the list view. Which I forget about every time since I am not using it anymore :wink: